docs/src/components/Theming/TokenList.tsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import { useTheme } from '@aws-amplify/ui-react';
+import { isDesignToken } from '@aws-amplify/ui';
+import {
+  SpaceBlock,
+  BorderWidthBlock,
+  FontFamilyBlock,
+  FontSizeBlock,
+  LineHeightBlock,
+  FontWeightBlock,
+  ColorBlock,
+  RadiusBlock,
+} from './TokenBlocks';
+
+type Namespaces =
```

Comment:
Just a follow up question, is it possible to derive this directly from the `Theme`? Just thinking that it would help give us some level of type safety/prevent deviation if a change is made in the `Theme` in the future

Reply:
I think I might leave cleaning up the Types further until post marketing launch. I was able to pull in some types directly from the @aws-amplify/ui package (like ColorBlock uses ColorValue, LineHeightBlock uses LineHeightValue) but ran into some missing exports when I tried to do this for others like [Tokens](https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/theme/tokens/index.ts#L25). I think we can find some ways around this though  (especially as we'd like to expand on showing component specific tokens at some point and having stronger typing will be nice for that).